compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -514,57 +525,87 @@ fn escape_stdout_stderr_string(s: &[u8]) -> String {
     })
 }
 
-const LLVM_DWP_EXECUTABLE: &'static str = "rust-llvm-dwp";
-
-/// Invoke `llvm-dwp` (shipped alongside rustc) to link `dwo` files from Split DWARF into a `dwp`
-/// file.
-fn link_dwarf_object<'a>(sess: &'a Session, executable_out_filename: &Path) {
-    info!("preparing dwp to {}.dwp", executable_out_filename.to_str().unwrap());
-
+/// Invoke `thorin` (rust implementation of a dwarf packaging utility) to link DWARF objects into a
```

Comment:
Nit: “Invoke” evokes an interpretation of starting an external process. “Use” would probably be more accurate.

Reply:
Oops, this is just a leftover from when we were invoking an external process. Fixed!